docs/src/pages/[platform]/theming/style-attributes/index.page.mdx

```diff
@@ -0,0 +1,322 @@
+---
+title: Component Style Attributes
+metaDescription: 'CSS Style Attributes that are available on every amplify-ui component'
+supportedFrameworks: react
+---
+
+import { Example, ExampleCode } from '@/components/Example';
+import { ComponentPropsToStylePropsMap, Card } from '@aws-amplify/ui-react';
+import {
+  BackgroundStyles,
+  BorderStyles,
+  ColorStyles,
+  DesignTokenTable,
+  FlexStyles,
+  GridStyles,
+  LayoutStyles,
+  MarginAndPaddingStyles,
+  PositionStyles,
+  ShadowStyles,
+  SizeStyles,
+  TypographyStyles,
+  OtherStyles,
+} from './examples';
+
+These style attributes are available on amplify-ui primitive components and can be used to modify the corresponding styles. This allows users to quickly make style adjustments without having to go through css or add a theme object.
```

Comment:
We should probably mention the CSS specificity of using style props. As in, style props will override default Amplify UI styling, class names, and the theme object (I think). 

Perhaps we should explain a good use case for using style props, other than just making quick changes (rapid prototyping) and flexibility?

Reply:
+1 to adding a note that says the prop styles always win specificity battles. 

I'm not sure what we would say regarding a good use case, it may be more of a personal preference with regards to DX.

Reply:
Text updated to reflect that the style props will take precedence over other forms of styling.